CMakeLists.txt

```diff
@@ -449,6 +451,11 @@ option(libiconv "enable linking against system libiconv" OFF)
 option(logging "build with logging" ON)
 option(build_tests "build tests" OFF)
 option(build_examples "build examples" OFF)
+option(python-bindings "build python bindings" OFF)
+
+if (build_tests OR (WIN32 AND python-bindings))
```

Comment:
The python bindings should not be using extra exported symbols. Is there an issue for this?

Reply:
On Windows the bindings library links to openssl symbols.